src/gen-xml.ts

```diff
@@ -962,6 +954,19 @@ function genXmlParagraphProperties (textObj: ISlideObject | TextProps, isDefault
 	return paragraphPropXml
 }
 
+function genXmlBulletMarginIndent (options: ObjectOptions | TextPropsOptions): string {
+	const bullet = typeof options.bullet === 'object' ? options.bullet : {}
+	const bulletMarL = valToPts(bullet.indent ?? DEF_BULLET_MARGIN)
+	const indentLevel = options.indentLevel ?? 0
+
+	if (!bullet.hangingIndentRatio) {
+		return ` marL="${indentLevel > 0 ? bulletMarL + bulletMarL * indentLevel : bulletMarL}" indent="-${bulletMarL}"`
+	}
```

Comment:
same behaviour as before. the value library users pass to a `bullet.indent` option is used as the hanging indent, and the actual "before text" indent in the way most people probably think of it (distance between left side of page and start of text) is 2x, 3x, 4x, the passed indent value. Note that it does not start at 1x, since if the `marL` and `indent` (before text, and hanging) indents are equal to each other (rather, like 9 and -9) then the bullet does not appear to be indented at all.